docs/releases/1.26-NOTES.md

```diff
@@ -86,3 +86,7 @@ CNIs, use the "cni" networking option instead.
 
 * kOps needs maintainers for Canal, Flannel, Kube-Router, and Spotinst to keep versions up to date and move the integration from experimental to stable.
 If no volunteers step up by the time kOps 1.27 is released, support will be phased out.
+
+# Other changes of note
+
+* Add unreachable route for pod IP on deletion option in Cilium [@anthonyhaussman](https://github.com/anthonyhaussman) [#15050](https://github.com/kubernetes/kops/pull/15050)
```

Comment:
Please move to the top, just before breaking changes.
```suggestion
* Added unreachable route for pod IP on deletion option in Cilium.
```

Reply:
Done :ok_hand: